util/errors/errors.go

```diff
@@ -17,7 +18,13 @@ func IsTransientErr(err error) bool {
 		return false
 	}
 	err = argoerrs.Cause(err)
-	return isExceededQuotaErr(err) || apierr.IsTooManyRequests(err) || isResourceQuotaConflictErr(err) || isTransientNetworkErr(err) || apierr.IsServerTimeout(err) || apierr.IsServiceUnavailable(err) || matchTransientErrPattern(err)
+	isTransient := isExceededQuotaErr(err) || apierr.IsTooManyRequests(err) || isResourceQuotaConflictErr(err) || isTransientNetworkErr(err) || apierr.IsServerTimeout(err) || apierr.IsServiceUnavailable(err) || matchTransientErrPattern(err)
+	if (isTransient) {
+		log.Infof("Transient error: %v", err)
+	} else {
+		log.Errorf("Fatal error: %v", err)
```

Comment:
```suggestion
		log.Errorf("Non-transient error: %v", err)
```

Technically not a fatal error :)